corehq/apps/domain/views/base.py

```diff
@@ -78,6 +80,23 @@ def select(request, do_not_redirect=False, next_view=None):
         return render(request, domain_select_template, additional_context)
 
 
+@login_required
+def accept_all_invitations(request):
+    def _invite(invitation, user):
+        user.add_as_web_user(invitation.domain, role=invitation.role,
+                             location_id=invitation.supply_point, program_id=invitation.program)
+        invitation.is_accepted = True
+        invitation.save()
+        send_confirmation_email(invitation)
+
+    user = request.couch_user
+    invites = Invitation.by_email(user.username)
+    for invitation in invites:
+        _invite(invitation, user)
+        messages.success(request,  _(f'You have been added to the "{invitation.domain}" project space.'))
```

Comment:
Is the typical use case likely to be 5 project spaces or 50? If it's the latter, might be better to just note the number of spaces, instead of a message for each.

Reply:
I suspect it varies widely. It seems to be 5-10 at a time for our current projects, because the ones that are in every domain handle it through mirroring. I could see the appeal of both options